apps/api/src/species/controller-project-species-one.ts

```diff
@@ -5,11 +5,14 @@ import { LocationProjectSpeciesFileModel } from '@rfcx-bio/node-common/dao/model
 import { ATTRIBUTES_TAXON_SPECIES_CALL, ATTRIBUTES_TAXON_SPECIES_PHOTO } from '@rfcx-bio/node-common/dao/types'
 
 import { getSequelize } from '@/_services/db'
+import { env } from '@/_services/env'
 import { BioNotFoundError } from '~/errors'
 import { isProtectedSpecies } from '~/security/protected-species'
 import { type Handler } from '../_services/api-helpers/types'
 import { assertPathParamsExist } from '../_services/validation'
 
+const CORE_API_BASE_URL = env.CORE_API_BASE_URL
+
```

Comment:
just use `env.CORE_API_BASE_URL` directly. no need to expose variable on this scope